virt/backup_restore/virt-backup-restore-overview.adoc

```diff
@@ -6,42 +6,46 @@ include::_attributes/common-attributes.adoc[]
-. Install the xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc#oadp-installing-dpa_installing-oadp-ocs[Data Protection Application] with the `kubevirt` and `openshift` xref:../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[plugins].
-. Back up virtual machines by creating a xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[`Backup` custom resource (CR)].
-. Restore the `Backup` CR by creating a xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[`Restore` CR].
+* Container Storage Interface (CSI) backups with DataMover
+
+The following storage options are excluded:
+
+* File system backup and restore
+
+* Volume snapshot backup and restore
+
+For more information, see xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc#oadp-backing-up-applications-restic-doc[Backing up applications with File System Backup: Kopia or Restic].
+====
+To install the {oadp-short} Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. See xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for details.
+
+include::modules/install-and-configure-oadp-kubevirt.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-[id="additional-resources_virt-backup-restore-overview"]
-== Additional resources
+.Additional resources
 
-* xref:../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features-plugins[OADP features and plugins]
-* xref:../../backup_and_restore/application_backup_and_restore/troubleshooting.adoc#troubleshooting[Troubleshooting]
-endif::openshift-rosa,openshift-dedicated[]
+* xref:../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[{oadp-short} plugins]
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[`Backup` custom resource (CR)]
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[`Restore` CR]
+* xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
 
-ifdef::openshift-rosa,openshift-dedicated[]
-Back up and restore virtual machines by using the OpenShift API for Data Protection (OADP).
+include::modules/oadp-installing-dpa-1-3.adoc[leveloffset=+1]
 
-.Prerequisites
+[IMPORTANT]
+====
+Red Hat supports using {VirtProductName} 4.14 or later with {oadp-short} 1.3.0 or later.
```

Comment:
Do we want to specify which version should go with which version?

Reply:
@weshayutin && @amastbau - is there any version relationship to be specified

Please and thanks

Reply:
@weshayutin && @amastbau - please can i give you a quick nudge on this so we can get it merge. OCP are picky about approvals

Reply:
Red Hat Supports using  {VirtProductName}  4.14 and 4.15 with 1.3.x